jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/OpenJ9DiagnosticsMXBeanImpl.java

```diff
@@ -28,13 +28,16 @@
 import javax.management.ObjectName;
 
 import com.ibm.jvm.Dump;
+import com.ibm.jvm.DumpPermission;
 import com.ibm.jvm.DumpConfigurationUnavailableException;
 import com.ibm.jvm.InvalidDumpOptionException;
 
 import com.ibm.jvm.Trace;
 import com.ibm.lang.management.OpenJ9DiagnosticsMXBean;
 
 public class OpenJ9DiagnosticsMXBeanImpl implements OpenJ9DiagnosticsMXBean {
+	private static final DumpPermission DUMP_PERMISSION = new DumpPermission();
```

Comment:
Checking security here is no longer required. Since my original comment, OpenJ9 has turned on the legacy security checks by default, in #979.